core/arch/riscv/kernel/spinlock.S

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Please use C style comments `/* SPDX-License-Identifier: BSD-2-Clause */` in assembly files.